frigate/http.py

```diff
@@ -198,7 +198,7 @@ def send_to_plus(id):
         return make_response(jsonify({"success": False, "message": message}), 404)
 
     # events from before the conversion to relative dimensions cant include annotations
-    if any(d > 1 for d in event.box):
+    if any(d > 1 for d in event.data["box"]):
```

Comment:
This should never be true. This check only applies when the box is stored in the old column. I think we can remove it and only submit examples when `data["box"]` is not null because these would have to have relative coordinates already.

In general, there is no need to handle the old events structure at all anymore.